include/XLink/XLinkPrivateDefines.h

```diff
@@ -134,8 +134,9 @@ typedef struct xLinkEventHeader_t{
     eventId_t           id;
     xLinkEventType_t    type;
     char                streamName[MAX_STREAM_NAME_LENGTH];
-    uint64_t            tsec;
     uint32_t            tnsec;
+    uint32_t            tsecLsb;
+    uint32_t            tsecMsb;
```

Comment:
I wouldn't add new fields in the middle of a struct/enum. Much better to add them at the end.

Reply:
The timestamp parameters use memory that was previously occupied by the bigger stream name buffer (for backward compatibility with bootloader) so they have to be between streamName and streamId.